src/admin-views/settings/tabs/display/display-qr-codes.php

```diff
@@ -36,13 +34,16 @@
 	];
 
 	$series = get_posts( $args );
-	foreach ( $series as $series ) {
-		$series_options[ $series->ID ] = $series->ID . ' - ' . $series->post_title;
+	if ( ! empty( $series ) ) {
+		$series_options = [];
+		foreach ( $series as $series ) {
+			$series_options[ $series->ID ] = $series->ID . ' - ' . $series->post_title;
```

Comment:
IMO, interpolation is easier to read.

```suggestion
			$series_options[ $series->ID ] = "{$series->ID} - {$series->post_title}";
```